addon/helpers/if-undefined.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function ifUndefined([condition, whenUndef, whenNotUndef, ...rest]) {
+  return condition === undefined ? whenUndef : whenNotUndef
+}
+
+export default Ember.Helper.helper(ifUndefined);
```

Comment:
If we decide this helper is necessary (see below) - I would still like it to come from `core`